### PR TITLE
Change question to "Não Divertido"

### PR DESCRIPTION
**Pull request by vitordullens**

### CODE REVIEW

File: Assets/Scenes/Teste 1.2/Perguntas2.unity [hunk 1]
```diff
@@ -1768,7 +1768,7 @@ MonoBehaviour:
     m_HorizontalOverflow: 0
     m_VerticalOverflow: 0
     m_LineSpacing: 1
-  m_Text: 'Muito Chato  '
+  m_Text: 'N\xE3o Divertido  '
```
Comment (VerasThiago) [inline Assets/Scenes/Teste 1.2/Perguntas2.unity:1771]: Is this right ?

Comment (vitordullens) [reply Assets/Scenes/Teste 1.2/Perguntas2.unity:1771]: based on other "não" in this same file, i think it is, but did not test it. 🙄

Comment (VerasThiago) [reply Assets/Scenes/Teste 1.2/Perguntas2.unity:1771]: It will only fail if we test it, so I think it's fine

**Review by VerasThiago — APPROVED** [review]
LGTM

**Pull request merged**